src/index.js

```diff
@@ -1,14 +1,50 @@
+/* eslint-disable no-unused-vars */
 import _ from 'lodash';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const todoItems = [];
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+function renderTodo(todo) {
+  const list = document.querySelector('.js-todo-list');
 
-  return element;
+  const isChecked = todo.checked ? 'done' : '';
+  const node = document.createElement('li');
+  node.setAttribute('class', `todo-item ${isChecked} border flex height pad-left`);
+  node.setAttribute('data-key', todo.index);
+  node.innerHTML = `
+    <div class="left flex">
+      <input class="check" id="${todo.index}" type="checkbox"/>
+      <label for="${todo.index}" class="tick js-tick"></label>
+      <span>${todo.text}</span>
+    </div>
+    <div class="right flex">
+      <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="ellipsis-v" class="svg-inline--fa move move-js-todo fa-ellipsis-v fa-w-6" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 192 512"><path fill="currentColor" d="M96 184c39.8 0 72 32.2 72 72s-32.2 72-72 72-72-32.2-72-72 32.2-72 72-72zM24 80c0 39.8 32.2 72 72 72s72-32.2 72-72S135.8 8 96 8 24 40.2 24 80zm0 352c0 39.8 32.2 72 72 72s72-32.2 72-72-32.2-72-72-72-72 32.2-72 72z"></path></svg>
+    </div>
+  `;
+
+  list.append(node);
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+function addTodo(text) {
+  const todo = {
+    text,
+    checked: false,
+    index: Date.now(),
```

Comment:
- [ ] It is good thinking to sort the array by the time, but according to the project requirements and the requirements of the next milestones of the same app, you'll have to sort the array by index number. For example, `index: 0`

Reply:
* done 👍